test006.c: 函数 `f' :
test006.c:6: エラー: `N' は宣言されていません.
test006.c:7: エラー: `M' は宣言されていません.
test006.c:8: エラー: `g' は宣言されていません.
test006.c:8: エラー: 函数ではない `g' を呼び出しました.
